index.html

```diff
@@ -109,165 +103,157 @@ <h2>Multi-Post<br> Stories</h2>
     <img src="Assets/images/tonic2.svg" alt="" class="image">
     <div class="left-card1-block">
       <div class="tonic">
-        <h2>Tonic</h2>
+        <h2 data-bss-hover-animate="rubberBand">Tonic</h2>
         <ul class="project-badges">
-          <li><a href="#">Canopy <img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
-          <li class="text-2"><a>Back&nbspEnd&nbspDev <img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
-          <li class="text-3"><a>2015</a></li>
+          <li><a href="#" class="text-1">CANOPY &nbsp;<img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
+          <li><a href="#" class="text-2">Back&nbsp;End&nbsp;Dev&nbsp; <img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
+          <li><a href="#" class="text-3">2015</a></li>
         </ul>
       </div>
-      <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups
+      <p class="project-description" data-bss-hover-animate="rubberBand">A daily selection of privately personalized reads; no accounts or sign-ups
         required.</p>
       <ul class="language-categories">
-        <li><a href="#">HTML</a></li>
-        <li><a href="#">css</a></li>
-        <li><a href="#">Javascript</a></li>
+        <li><a href="#" data-bss-hover-animate="pulse">HTML</a></li>
+        <li><a href="#" data-bss-hover-animate="pulse">css</a></li>
+        <li><a href="#" data-bss-hover-animate="pulse">Javascript</a></li>
       </ul>
-      <button class="submit-btn" type="submit">See Project</button>
+      <button class="submit-btn" type="submit" data-bss-hover-animate="pulse">See Project</button>
     </div>
   </div>
 
   <div class="card">
     <img src="Assets/images/multistories2.svg" alt="" class="image">
     <div class="left-card1-block">
       <div class="tonic">
-        <h2>Multi-Post<br> Stories</h2>
+        <h2 data-bss-hover-animate="rubberBand">Multi-Post<br> Stories</h2>
         <ul class="project-badges">
-          <li><a href="#">Canopy <img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
-          <li class="text-2"><a>Back&nbspEnd&nbspDev <img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
-          <li class="text-3"><a>2015</a></li>
+          <li><a href="#" class="text-1">CANOPY &nbsp;<img src="Assets/images/counter-images.png" alt="" class="counter-image"></a></li>
```

Comment:
- [ ] Kindly consider adding a `.` in front of your image path as that will tell the `src` prop to look for the images in the current directory. Because you haven't specified where the `src` prop to fetch the photos from they are currently not loading in your deployed app. Please consider fixing this issue as it will help create a better user experience. See below for more details:
![IMG_0039](https://github.com/lucy-sees/Portfolio/assets/107192076/0a1d4a55-1d17-43e2-8ec1-38b449d3c5ce)